internal/controller/controllers/bmh_agent_controller.go

```diff
@@ -238,7 +253,7 @@ func (r *BMACReconciler) Reconcile(origCtx context.Context, req ctrl.Request) (c
 	}
 
 	if result.Stop(ctx) {
-		log.Debugf("Stopping BMAC reconcile after reconcileAgentSpec")
+		log.Infof("Stopping BMAC reconcile after reconcileAgentSpec")
```

Comment:
why? is this done just to inspect CI logs?

Reply:
I added a hold. This is just for debugging purposes